.github/workflows/release-docker.yml

```diff
@@ -56,14 +56,11 @@ jobs:
     outputs:
       fully_qualified_tags: ${{ steps.tags.outputs.fully_qualified_tags }}
     steps:
-      - name: Check out the repo
-        uses: actions/checkout@v4
```

Comment:
We don't use this since it's checking out the repo relative to the workflow it's triggered from.